examples/apps/aml/web-client/README.md

```diff
@@ -0,0 +1,46 @@
+# Santander AML DEMO SGX WEB CLIENT
```

Comment:
These are useful instructions, but hard to find.  Can you link to this file from `examples/apps/aml/README.md`?